src/components/Visualization/Visualization.js

```diff
@@ -176,6 +177,20 @@ UnconnectedVisualization.propTypes = {
     onLoadingStart: PropTypes.func,
 }
 
+UnconnectedVisualization.contextType = ChartContext
+
+/* Setting these contextTypes is required for Jest/Enzyme
```

Comment:
It would be interesting to see if converting the tests to testing-library would resolve this problem, or if it is jest that is the problem?

Reply:
I've refactored this code to be more in line with the approach @edoardo took when removing d2 and now I don't need set these contextTypes anymore.